pre-commit.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+echo
+echo 'linting python3 code:'
+echo '====================='
+flake8 --max-line-length=119 source/
```

Comment:
`max-line-length` belongs in a [per-project configuration](http://flake8.readthedocs.org/en/stable/config.html#per-project), so that you work with the same settings when just invoking `flake8` on the command line.